src/rc/start-stop-daemon.c

```diff
@@ -712,6 +712,17 @@ start_stop_daemon(int argc, char **argv)
 		if (sscanf(tmp, "%d", &nicelevel) != 1)
 			eerror("%s: invalid nice level `%s' (SSD_NICELEVEL)",
 			    applet, tmp);
+		if ((tmp = getenv("SSD_IONICELEVEL"))) {
```

Comment:
If you expand the context, you'll see that line 712 is indented as far as it is because it's the child of an if statement (line 711).  Since line 715 is not part of that if statement, it should have the same indentation as the if statement.